### DISCUSSION TITLE
DataCite 4.4 new fields for publications and conferences

### DISCUSSION THREAD

**Discussion by lnielsen** [inveniosoftware/product-rdm#136] [Ideas]
Zenodo has fields for publications and conferences. We'll need either custom fields or extending the core data model to support the new DataCite 4.4 fields 

See example of problem in inveniosoftware/product-rdm#80

**Comment by ntarocco** [comment]
Duplication of https://github.com/inveniosoftware/product-rdm/discussions/150
